releases/7.6.0.md

```diff
@@ -302,7 +303,9 @@ Minor incompatibilities:
     3. If it exists, remove `${project_name}/webpack/webpack-user-config.js`
 
 10. In settings.py:
-    1. Remove `"compressor"` from `INSTALLED_APPS`
+    1. In `INSTALLED_APPS`:
+        - remove `"compressor"`
+        - add `"arches.apps"`
```

Comment:
There's a fanciness judgment call here ✨:

if we don't like having to add "arches.apps" to be able to let django find "arches.apps.templates" by default, we can create a custom loader that does something fancy like: look for every app except core arches under /templates, but look for core arches under app/templates. I just paused for input before adding such an invasive workaround.

Reply:
Another option is to just keep the approach before we discussed on Tuesday at core standup and just admit a little indirection.